Update maintenance-policy.md (#1579)
spectrocloud/librarium: docs/docs-content/integrations/maintenance-policy.md

tags: ["packs", "deprecation"]
---

> Palette supports two pack categories: *Infrastructure* and *Add-on*. Infrastructure packs, which we often source from third parties, are infrastructure related or have prominence in container-based environments. Packs are used to create layers in cluster profiles. Infrastructure packs are grouped as follows:

Reviewer: ```suggestion
Palette supports two pack categories: *Infrastructure* and *Add-on*. Infrastructure packs are often sourced from third parties and are infrastructure-related or support critical container-based environments. Packs are used to create layers in cluster profiles. Infrastructure packs are grouped as follows:
```
